src/main/webapp/resources/js/contexts/AnalysisContext.js

```diff
@@ -44,6 +44,30 @@ const AnalysisContext = React.createContext(initialContext);
 function AnalysisProvider(props) {
   const [analysisContext, setAnalysisContext] = useState(initialContext);
 
+  // Update the analysis details that
+  // are required to display the progression
+  // using polling
+  useEffect(() => {
+      const interval = setInterval(() => {
+        getUpdatedDetails(analysisContext.analysis.identifier).then(res => {
+          setAnalysisContext(analysisContext => {
+            return {
```

Comment:
This is happening even if nothing changes?   Shouldn't you do a check first to see if an update is needed?

Reply:
Updated in c64a856128d212fdda1bcab36dbf56b78cef8399